src/viam/ur/module/ur_arm.cpp

```diff
@@ -372,6 +375,18 @@ void URArm::configure_(const std::unique_lock<std::shared_mutex>& lock, const De
     current_state_->host = find_config_attribute<std::string>(cfg, "host");
     current_state_->speed.store(degrees_to_radians(find_config_attribute<double>(cfg, "speed_degs_per_sec")));
     current_state_->acceleration.store(degrees_to_radians(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2")));
+    try {
```

Comment:
We do actually have a config validator, in `validate_config_`. Perhaps this checking belongs there. I'm not sure why it has a trailing `_`, in case you want to clean that up while you are there.

Reply:
Agreed. I filed https://viam.atlassian.net/browse/RSDK-11547.